File: src/CrowdfindingApp.Client/src/app/components/project-card/ProjectCard.ts
import { ProjectStatusEnum } from 'src/app/models/enums/ProjectStatus';

export class ProjectCard {

constructor(
  public status: ProjectStatusEnum,
  public name: string,
  public description: string,
  public categoryName: string,
  public categoryId: string,
  public imgPath: string,
  public purpose: number,
  public currentResult: number,
  public id: string) {}
}

File: src/CrowdfundingApp.Common.Core/DataTransfers/Projects/ProjectCard.cs

using CrowdfindingApp.Common.Enums;

namespace CrowdfindingApp.Common.Core.DataTransfers.Projects
{
    public class ProjectCard
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string CategoryName { get; set; }
        public string CategoryId { get; set; }
        public string ImgPath { get; set; }
        public decimal? Purpose { get; set; }
        public decimal? CurrentResult { get; set; }
        public ProjectStatus Status { get; set; }
        public string RestTimeToEnd { get; set; }
    }
}
